libevmasm/ConstantOptimiser.h

```diff
@@ -100,7 +103,7 @@ class LiteralMethod: public ConstantOptimisationMethod
 	explicit LiteralMethod(Params const& _params, u256 const& _value):
 		ConstantOptimisationMethod(_params, _value) {}
 	virtual bigint gasNeeded() override;
```

Comment:
I don't understand what this method returns.  Is it the cost in the deployment code or in the runtime code, and moreover, is it the cost before the optimization or after the optimization?